website/docs/en/config/plugins.mdx

```diff
@@ -121,4 +121,8 @@ export default defineConfig({
 });
 ```
 
+:::tip
+When using unplugin transform hook, you need to use transformInclude hook to exclude the html module. Transform hook will destroy the html-rspack-plugin plug-in to generate html.
```

Comment:
```suggestion
When using the transform hook in unplugin, please use the `transformInclude` hook to match the specified module. When the transform hook matches the `.html` module, it will replace the default EJS conversion feature of the html-rspack-plugin.
```